.github/actions/make_init/action.yml

```diff
@@ -0,0 +1,53 @@
+# https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#adding-an-action-from-the-same-repository
+runs:
+  using: composite
+  steps:
+    - name: "Install NVM, Node.js, and Yarn"
+      uses: actions/setup-node@v3
+      with:
+        node-version: lts/gallium
```

Comment:
Do you consider creating a `.nvmrc` file to avoid hardcoding this version in action?